pom.xml

```diff
@@ -282,6 +282,7 @@
           <configuration>
             <compilerArgs>
               <arg>-Aorg.jboss.logging.tools.addGeneratedAnnotation=false</arg>
+              <arg>-proc:full</arg>
```

Comment:
Using this argument I could see some errors in byteman tests (e.g. '-mvn clean install -rf :jta -Parq -Dtest=RecoverySuspendTest -Dmaven.compiler.proc=full' using jdk 25). 
FYI The [failure](https://github.com/tomjenkinson/narayana/actions/runs/18750641114/job/53489052097#step:6:4785) i[s not present ](https://github.com/marcosgopen/workflows/actions/runs/18776566622/job/53572489476)when adding the annotation processor in the maven-compiler-plugin instead (see https://github.com/jbosstm/narayana/pull/2418/files#diff-09e4325cb9f0ddb0c933f2fa677782150ca9a54790551e9c55333bc5bdae5448R154-R165). That is also the recommended solution by James.